README.md

```diff
@@ -175,6 +175,13 @@ A special thanks to:
 - [Start Bootstrap](https://startbootstrap.com/theme/resume) for amazing themes!
 - [GitHub actions](https://docs.github.com/en/free-pro-team@latest/actions) docs, it's the best guide!
 
+
+### Our Valuable Contributors 👩‍💻👨‍💻 :
```

Comment:
consistent heading levels are required. This is level 3 and the rest of them are 2(##). Also, increase the size of these thumbnails.

Reply:
okay I'll modify this